src/bp-forums/templates/default/bbpress/form-topic.php

```diff
@@ -90,9 +90,26 @@
 
 						<?php do_action( 'bbp_theme_before_topic_form_tags' ); ?>
 
+						<?php
+						$get_topic_id = bbp_get_topic_id();
+						$get_the_tags = isset($get_topic_id) && !empty($get_topic_id) ? bbp_get_topic_tag_names($get_topic_id) : '';
+						?>
+
 						<p>
-							<input type="hidden" value="" name="bbp_topic_tags" class="bbp_topic_tags" id="bbp_topic_tags" >
-							<select name="bbp_topic_tags_dropdown[]" class="bbp_topic_tags_dropdown" id="bbp_topic_tags_dropdown" placeholder="<?php esc_html_e( 'Type one or more tag, comma separated', 'buddyboss' ); ?>" autocomplete="off" multiple="multiple" style="width: 100%" tabindex="<?php bbp_tab_index(); ?>"></select>
+							<input type="hidden" value="<?php echo $get_the_tags; ?>" name="bbp_topic_tags" class="bbp_topic_tags" id="bbp_topic_tags" >
```

Comment:
@sadrul Can we use escaping function for `get_the_tags`
```suggestion
							<input type="hidden" value="<?php echo esc_attr( $get_the_tags ); ?>" name="bbp_topic_tags" class="bbp_topic_tags" id="bbp_topic_tags" >
```